userspace/libsinsp/event.h

```diff
@@ -31,11 +31,11 @@ typedef class sinsp_threadinfo sinsp_threadinfo;
 ///////////////////////////////////////////////////////////////////////////////
 typedef enum filtercheck_field_flags
 {
-	EPF_NONE = 0,
```

Comment:
it looks they are used right now only exclusively but using `&`, so to have more space for the future makes sense to me to put them as bitflags.